docs/scenarios/jdk_8_to_17_migration/README.md

```diff
@@ -263,38 +290,41 @@ Expand the incidents for `Library.java` and you will see three incidents
 | **`javax.servlet` → `jakarta.servlet`** | Line 8       | Update import              |
 | **Deprecated `BASE64Encoder`**          | Line 7       | Use `Base64.getEncoder()`  |
 
-![incidents_library.png](images/incidents_library.png)
+![library-issues.png](images/library_java_issues.png)
 
 #### MD5 is outdated
 
 #### Initiating the Resolution
 
 - Click on the **tool icon** next to the first issue (MD5 hashing).
 
-![get_solution.png](images/get_solution.png)
+![first-incident.png](images/first_incident.png)
 
 - This opens the resolution details panel
-  ![solution_details.png](images/solution_details.png)
+  ![incident_1.png](images/incident_1.png)
+  ![kai-results-1.png](images/kai_results_1.png)
 
 #### Requesting a Fix
 
 - The **resolution process starts**, where Kai interacts with the **LLM**.
 - Once Kai completes processing, it suggests modifications for **Library.java**.
-  ![requesting_fix_1.png](images/requesting_fix_1.png)
-  ![requesting_fix_2.png](images/requesting_fix_2.png)
+  ![kai-results-1.png](images/kai_results_1.png)
+  ![kai-results-2.png](images/kai_results_2.png)
+  ![kai-results-3.png](images/kai_results_3.png)
 
 #### Reviewing Suggested Changes
 
 - The IDE highlights differences between the **current** and **suggested** versions.
 - The **right panel** displays:
   - **MD5 is replaced with SHA-256** for secure hashing.
   - **`BASE64Encoder` is replaced with `Base64.getEncoder()`** from `java.util.Base64`, which is the standard modern API.
-    ![file_diff_view.png](images/file_diff_view.png)
+    ![suggested-changes.png](images/3.1_changes.png)
```

Comment:
capture the md5 changes as that is the solution for the custom rule violation. example - https://github.com/konveyor/kai/blob/main/docs/scenarios/jdk_8_to_17_migration/images/file_diff_view.png